src/main/java/frc/robot/RobotContainer.java

```diff
@@ -83,6 +95,28 @@ private Command doNothing() {
   private void configureBindings() {
     new Trigger(() -> triggerPressed())
         .whileTrue(new BasicDriveCommand(m_robotDrive, m_driverController));
+
+    new Trigger(this::getIntakeButton).whileTrue(new RunIntake(m_intake));
+    new Trigger(this::getReverseIntakeButton).whileTrue(new RunIntakeReversed(m_intake));
```

Comment:
Do we still need to reverse the intake for competition?

Reply:
It's for if the note gets stuck; we're hoping we don't need it.